pkgs/tools/text/boxes/default.nix

```
@@ -0,0 +1,45 @@
+{ stdenv, fetchFromGitHub, bison, flex }:
+
+stdenv.mkDerivation rec {
+  pname = "boxes";
+  version = "1.3";
+
+  src = fetchFromGitHub {
+    owner = "ascii-boxes";
+    repo = "boxes";
+    rev = "v${version}";
+    sha256 = "0b12rsynrmkldlwcb62drk33kk0aqwbj10mq5y5x3hjf626gjwsi";
+  };
+
+  # Building instructions:
+  # https://boxes.thomasjensen.com/build.html#building-on-linux--unix
+  buildInputs = [ bison flex ];
+
+  dontConfigure = true;
+
+  # Makefile references a system wide config file in '/usr/share'. Instead, we
+  # move it within the store by default.
+  preBuild = ''
+    sed -i "/^GLOBALCONF = \/usr\/share\/boxes$/c\GLOBALCONF=$out/share/boxes-config" Makefile
```

If we use `substituteInPlace` we don't have to worry about escaping
```suggestion
    substituteInPlace Makefile \
      --replace "GLOBALCONF = /usr/share/boxes" "GLOBALCONF=${placeholder "out"}/share/boxes/boxes-config"
```